backend/src/application/controllers/rating/create-rating-controller.ts

```diff
@@ -3,6 +3,10 @@ import AppDataSource from '@/infra/repositories/mysql/data-source'
 import { Rating, Ticket, User } from '@/infra/repositories/mysql/entities'
```

Comment:
Esses imports deixaram de ser utilizados.

```suggestion
import { Rating } from '@/infra/repositories/mysql/entities'
```